package application.exceptions;

public class LoginAlreadyExists extends Throwable {

}
